scripts/install/lazylibrarian.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+# LazyLibrarian install script for swizzin
+# Author: Aethaeran
+
+##########################################################################
+# References
+##########################################################################
+
+# https://github.com/Aethaeran/swizzin/blob/feat/jdownloader/scripts/install/jdownloader.sh
+# https://lazylibrarian.gitlab.io/install/
+# https://www.reddit.com/r/LazyLibrarian/comments/jw5e22/lazylibrarian_calibre_calibreweb_booksonic/
+# https://github.com/lazylibrarian/LazyLibrarian
+# https://gitlab.com/LazyLibrarian/LazyLibrarian
+# https://lazylibrarian.gitlab.io/config_commandline/
+# https://github.com/swizzin/swizzin/pull/743
+# https://lazylibrarian.gitlab.io/config_users/
+# https://lazylibrarian.gitlab.io/config_downloaders/
+# https://www.reddit.com/r/LazyLibrarian/
+# https://lazylibrarian.gitlab.io/api/
+
+##########################################################################
+# Import Sources
+##########################################################################
+
+. /etc/swizzin/sources/functions/color_echo
+. /etc/swizzin/sources/functions/pyenv
+. /etc/swizzin/sources/functions/users
+. /etc/swizzin/sources/functions/utils
+
+##########################################################################
+# Variables
+##########################################################################
+
+app_name="lazylibrarian"
+pretty_name="LazyLibrarian"
+default_port="5299"
+master=$(_get_master_username)
+data_dir="/home/$master/.config/lazylibrarian"
+app_dir="/opt/$app_name"
+venv_dir="/opt/.venv/$app_name"
+
+##########################################################################
+# Functions
+##########################################################################
+
+function _dependencies() {
+    echo_info "Installing dependencies for $pretty_name..."
+
+    echo_progress_start "Creating $pretty_name venv"
+    mkdir -p "$venv_dir"
+    python3_venv "$master" "$app_name"
+    echo_progress_done
+
+    echo_progress_start "Installing python dependencies to venv"
+    # shellcheck disable=2154           # log variable is inherited from box itself.
+    "$venv_dir/bin/pip" install --upgrade pip >>"${log}" 2>&1 # Upgrade PIP
+    PIP='apprise pyOpenSSL'
+    # shellcheck disable=2086           # We want the $PIP variable to expand. So 2086's warning is invalid here.
+    "$venv_dir/bin/pip" install $PIP >>"${log}" 2>&1
```

Comment:
all cap variables are "reserved" for environment variables, please use something like `pipreqs` or something. Could also move it to the very top

OR, just skip the variable altogether if it's used just once

Reply:
Can change. I copied this variable naming convention from another install script. Would you like me to make a quick PR changing it in other scripts as well?

Reply:
Changed `PIP` to `pip_reqs`. Moved its declaration up with the rest of the variables.